public/content/translations/pl/refi/index.md

```diff
@@ -14,21 +14,23 @@ summaryPoint3: Narzędzie do drastycznego skalowania aktywów przynoszących kor
 
 ## Czym jest ReFi? {#what-is-refi}
 
-**Finanse regeneracyjne (ReFi)** to zestaw narzędzi i pomysłów opartych na blockchainach, których celem jest tworzenie gospodarek, które są regeneracyjne, a nie wydobywcze lub eksploatacyjne. Ostatecznie systemy wydobywcze wyczerpią dostępne zasoby i upadną; bez mechanizmów regeneracyjnych brakuje im odporności. ReFi działa w oparciu o założenie, że tworzenie wartości pieniężnej musi być oddzielone od niezrównoważonego wydobycia zasobów z naszej planety i społeczności.
+**Regenerative finance (ReFi)** is a set of tools and ideas built on top of [blockchains](/glossary/#blockchain), that aim to create economies which are regenerative, rather than extractive or exploitative. Ostatecznie systemy wydobywcze wyczerpią dostępne zasoby i upadną; bez mechanizmów regeneracyjnych brakuje im odporności. ReFi działa w oparciu o założenie, że tworzenie wartości pieniężnej musi być oddzielone od niezrównoważonego wydobycia zasobów z naszej planety i społeczności.
```

Comment:
A lot of English regressions?

Was this brought in using the script @wackerow? Looks like it isn't ready for upload yet

<img width="1396" alt="Screenshot 2024-04-23 at 12 29 03" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/1e4505ed-bbae-4ab9-a404-eb9efec33669">

Reply:
cc: @lukassim

Reply:
Yes Paul and I spoke about this, we uploaded these translations (manually, not using the script) from the latest build, not the one from April 2nd, before we updated the source...
I think we're still fine to pull them in as they'll get updated soon anyway, but we could also revert these and pull in the translations from the April 2nd download insted
Thoughts @minimalsm?